posting/index.go

```diff
@@ -662,19 +663,21 @@ func (r *rebuilder) Run(ctx context.Context) error {
 		return err
 	}
 	glog.V(1).Infof("Rebuilding index for predicate %s: building temp index took: %v\n",
-		r.attr, time.Since(start))
+		x.FormatNsAttr(r.attr), time.Since(start))
```

Comment:
here as well. Print namespace everywhere. Without the namespace, we wouldn't know which namespace is doing this operation.

Reply:
It is already printing namespace information. 
`FormatNsAttr(attr)` formats them to this format: `x-predicate` 

predicate `name` from namespace `1` will be printed as `1-name` rather than current thing which is `/000/000/000/000/000/000/000/001name`

Reply:
My bad. I didn't realize the formatNSAttr was adding the namespace as well.